recipe/meta.yaml

```diff
@@ -10,21 +10,21 @@ source:
   sha256: 5598c0a1a6eeb577bc271cac59983c89eb4c1beb1b7e75ba50068e2dc0c038f2
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win or py27]
```

Comment:
You'll want to add something like `or py == 38` here and remove `<3.8` from the host and run sections.

Reply:
Ahh, I wasn't sure how to do this- thanks @dpryan79